components/extractors/other/telemetry-data/telemetry-data.md

```diff
@@ -41,11 +41,14 @@ This table shows snapshots of buckets in Storage.
 | `snapshot_date` (PK) | Date of the data snapshot | `2020-06-30` |
 | `stage` | Storage stage of the bucket | `in` |
 | `bucket` | Name of the bucket | `c-instagram` |
+| `bucket_display_name` | Name of the bucket | `instagram` |
```

Comment:
@lucie-krasna Tady by mělo být z docs patrné, že to je jméno, které můžeš změnit na úrovní UI. Možná že `bucket` má být něco jako "Default name of the bucket".